application/values.yaml

```diff
@@ -909,7 +909,8 @@ certificate:
   #    - Stockholm
   # -- (string) Common name as specified on the DER encoded CSR.
```

Comment:
Please add the documentation in this value file, the readme is generated by make build-docs.